README.md

```diff
@@ -1 +1,146 @@
+
+### This project goes live [here](https://yemisirach.github.io/Awsome-book-ES6/) 🚀 .
+
+<br>
+
+<!-- DEPLOYMENT -->
+<!-- ### Have a look at the project's video presentation [here](https://www.loom.com/share/1f7de2b9345a4d3988c6327952899d30) 📽️ . -->
+
+<br>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome Books ES6] <a name="about-project"></a>
+
+**Awesome Books** is a pair programming project for saving book details like an online library.
+
+## 🛠 Built With <a name="built-with"></a>
+
+-Client
+
+- **This Project has a Readme.md file**
+- **This Project has a gitignore file**
+- **This projects follows all Microverse Linters**
+- **This project takes user inputs of Book title and Authors**
+- **This project displays the input for Book details dynamically on the DOM**
+- **This project saves the Book details insode the Browser**
+
+<a href="https://yemisirach.github.io/Awsome-book-ES6/">Live Demo link</a>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to copy the link of this Repository.
+
+```sh
+ https://github.com/Yemisirach/Awesome_books.git
+```
+
+<br>
+
+### Setup
+
+Clone this repository to your desired directory using the command:
+
+```sh
+  cd your-folder
+  git clone https://github.com/Yemisirach/Awesome_books.git
+```
+
+<br>
+
+### Usage
+
+Kindly go ahead and give this project a massage but do not forget to give a detailed description in your pull request.
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Yemisirach
+
+- GitHub: [@githubhandle](https://github.com/Yemisirach)
+- Twitter: [@twitterhandle](https://twitter.com/TamiratYemisrach)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/yemisirach)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[CSS styling]**
+- [ ] **[Multi Pages]**
+- [ ] **[Server Storage]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly support by suggesting changed by forking and pull requests
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank everyone who inspired us to learn programming. Regards to the whole Microverse team, Peers, and family supporting.
+
+<!-- <br>A massive Thank You to [Cindy Shin](https://www.behance.net/adagio07) who is the original designer of this [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./license) licensed.
```

Comment:
- [ ] Great job working on your documentation! 💯 However, please make sure that all links are working correctly, at this moment, this `MIT` link is broken because the actual license file you have in your repo is in caps: `LICENSE`. 🤓 🔧 

```suggestion
This project is [MIT](./LICENSE) licensed.
```